committees.md

```diff
@@ -1,14 +1,39 @@
-# Committees and Responsibilities
-## PyCon Chair
+# Volunteer Committee Head
 
-The conference chair is the main person in charge. He’s responsible for setting the budget, monitoring expenses and general oversight of all other committees. He works together with the Python.PH Board of Trustees for decision-making and to sort-out stuff behind the scenes.
+As a committee leader, you are responsible for coordinating to your committee volunteers, identifying tasks that need to be done, proper delegation and providing clear instructions to them.
 
-## Committee Heads
+## Preparation for PyCon
+- Identify the actionable items for your team.
+- Assign each volunteer to an action
+- Be sure to know what to do during an emergency. Coordinate with the security committee.
 
-As a committee head, you are responsible for coordinating to your committee members (volunteers for your committee), identifying tasks that need to be done, proper delegation and providing clear instructions to them.
+## During the PyCon
+- Communicate with your team and the PyCon Chair.
+- Be pro-active in any potential problems. If you identify one, fix it before it becomes an issue
+- While the Chair has the final decision, during an emergency or if necessarily, you have to make a decision on the spot.
 
-Some volunteers are shy at first. It might be a good idea to meet them for coffee/dinner so you can give them instructions in person and so that they'll feel more comfortable talking to you.
+## After the PyCon
+- Relax. Managing a conference is a hard and fulfilling endeavor. You deserve to relax
 
-We also found based on experience that one of the best ways to lead an open source community is to lead by example.
 
-Lastly, you rock! Thanks so much for stepping-up! :)
+# Tips for managing volunteers
+
+Each leader has their own management style. But for first-timers or those who are unsure of how to do their roles, we recommend the following:
+
+- **Prepare `goals` for your committee.** This is in addition to the goals the PyCon Chair assigns to you. These goals should be meant to push your committee to a higher bar.
+
+```This is a psychological trick where you set a target that is beyond the minimum. This will give you greater focus without compromising the minimum requirement.
```

Comment:
This is supposed to be viewed at gitbook (https://www.gitbook.com/book/pythonph/pycon-organizer-manual/details), so the backticks should render correctly. I'll wait for @marksteve to merge the pr and see the output.

This is my first time publishing to gitbook, so I'm sure there's a better workflow. I'll figure that out as I jump. ;)